dm in cloud: add billing for dm in cloud (#11635)
pingcap/docs: tidb-cloud/tidb-cloud-billing-dm.md

---
title: Data Migration Billing

> summary: Learn about billing for Data Migration in TiDB Cloud.

Reviewer: Learn about billing for Data Migration on TiDB Cloud

---

# Data Migration Billing

> This document describes the billing for Data Migration in TiDB Cloud.

Reviewer: This document describes the billing for Data Migration on TiDB Cloud.

## Specifications for Data Migration

TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs). When you create a data migration job, you can select an appropriate specification. You will be charged for these Data Migration RCUs.

> TiDB Cloud measures incremental data migration in rows/s. This unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. It also means if the upstream executes INSERT, UPDATE, or DELETE statements within 10000 rows, the Data Migration job can replicate the 10000 rows to the downstream within 1 second.

Reviewer: statements within 10000 rows/s

All the following performance values are maximum performances. Note that the performance values are for reference only and might vary in different scenarios.

The following table lists the specifications for Data Migration:

| Specification (RCUs)| Full data migration | Incremental data migration (rows/s)|
|---------------|---------------------|----------------------------|
| 2   | 25 MiB/s | 10000  |
| 4   | 35 MiB/s | 20000 |
| 8   | 40 MiB/s | 40000 |
| 16  | 45 MiB/s | 80000 |

## Price

The following table lists the price of TiDB Cloud for each Data Migration RCU:

| Cloud provider | Region                      | RCU price ($/hr) |
|----------------|-----------------------------|------------------|
| AWS            | Oregon (us-west-2)          |          $0.1307 |
| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |

> 

Reviewer: other regions on AWS also will be released at 27,DEC

Author: OK  added all.

---
title: Data Migration Billing
summary: Learn about billing for Data Migration in TiDB Cloud.
---

# Data Migration Billing

This document describes the billing for Data Migration in TiDB Cloud.

## Specifications for Data Migration

> 

Author: ```suggestion

All the following performance values are maximum performances. Note that the performance values are for reference only and will vary in different scenarios.
```

> | Specification | Full data migration | Incremental data migration |

Reviewer: Specification add the unit of RCUs ，such as 2 RCUs

|---------------|---------------------|----------------------------|

> | 2  | 25 MiB/s | 10 k [^*] |

Reviewer: add the unit of  Rows/s,such as 10k rows/s

| 4  | 35 MiB/s | 20 k |
| 8  | 40 MiB/s | 40 k |
| 16 | 45 MiB/s | 80 k |

> [^*]: this unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. For example, 10 k means 10000 rows per second.

Reviewer: 10 k means 10000 rows per second and also means if the upstream insert \update\delete within 10000 rows,the dm job can sync the 10000 rows to the downstream within 1 second.

> [^*]: this unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. For example, 10 k means 10000 rows per second.

Author: ```suggestion
| Specification (RCUs) [^1]| Full data migration | Incremental data migration (rows/s) [^2]|
|---------------|---------------------|----------------------------|
| 2   | 25 MiB/s | 10000  |
| 4   | 35 MiB/s | 20000 |
| 8   | 40 MiB/s | 40000 |
| 16 | 45 MiB/s | 80000 |

[^1]: TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs).
[^2]: this unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. It also means if the upstream executes INSERT, UPDATE, or DELETE statements within 10000 rows, the Data Migration job can replicate the 10000 rows to the downstream within 1 second.
```

> Maximum performance:

Reviewer: All the performance value are maximum performance instead of Maximum performance

> Maximum performance:
> 
> - Full data migration: 20 MiB/s
> - Incremental data migration: 5000 rows/s

Author: ```suggestion
```

> 

Reviewer: delete the following sentences

Full data migration: 20 MiB/s
Incremental data migration: 5000 rows/s

> Note that the performance values are for reference only and will vary in different scenarios.

Reviewer: delete the scentence
Note that the performance values are for reference only and will vary in different scenarios.

> Note that the performance values are for reference only and will vary in different scenarios.

Author: ```suggestion
```

> 

Author: ```suggestion
```

## Price

> The following table lists the price of TiDB Cloud for each TiCDC RCUs:

Reviewer: Not TiCDC RCUs but Data Migration RCUs

> The following table lists the price of TiDB Cloud for each TiCDC RCUs:
> 
> | Cloud provider | Region                      | RCU price ($/hr) |
> |----------------|-----------------------------|------------------|
> | AWS            | Oregon (us-west-2)          |          $0.1307 |
> | AWS            | N. Virginia (us-east-1)     |          $0.1307 |
> | AWS            | Mumbai (ap-south-1)         |          $0.1393 |
> | AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
> | AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
> | AWS            | Frankfurt (eu-central-1)    |          $0.1564 |

Author: ```suggestion
The following table lists the price of TiDB Cloud for each TiCDC RCUs:

| Cloud provider | Region                      | RCU price ($/hr) |
|----------------|-----------------------------|------------------|
| AWS            | Oregon (us-west-2)          |          $0.1307 |
| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
```

---
title: Data Migration Billing
summary: Learn about billing for Data Migration in TiDB Cloud.
---

# Data Migration Billing

This document describes the billing for Data Migration in TiDB Cloud.

## Specifications for Data Migration

TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs). When you create a data migration job, you can select an appropriate specification. The higher the RCU, the better the migration performance. You will be charged for these Data Migration RCUs.

> The following table lists the specifications for Data Migration.

Author: ```suggestion
The following table lists the specifications and corresponding performances for Data Migration.
```

TiDB Cloud measures full data migration in MiB/s. This unit indicates how much data (in MiB) per second is migrated by the Data Migration job.

Data Migration job measures incremental data migration performance in rows/s. This unit indicates the number of rows be migrated to target database per second. For example, if the upstream database executes INSERT, UPDATE, or DELETE statements of 10000 rows in about 1 second, the Data Migration job of corresponding specification can replicate the 10000 rows to the downstream in about 1 second.

## Price

The following table lists the price of TiDB Cloud for each Data Migration RCU:

| Cloud provider | Region                      | RCU price ($/hr) |
|----------------|-----------------------------|------------------|
| AWS            | Oregon (us-west-2)          |          $0.1307 |
| AWS            | N. Virginia (us-east-1)     |          $0.1307 |
| AWS            | Mumbai (ap-south-1)         |          $0.1393 |
| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
| AWS            | Frankfurt (eu-central-1)    |          $0.1564 |

> The Data Migration job is in the same region as the target TiDB Cluster.

Reviewer 2: ```suggestion
The Data Migration job is in the same region as the target TiDB cluster.
```

- If the source database and the TiDB cluster are not in the same region, cross-region traffic charges are incurred when the Data Migration job collects data from the source database.

    ![Cross-region traffic charges](/media/tidb-cloud/dm-billing-cross-region-fees.png)

- If the source database and the TiDB cluster are in the same region but in different AZs, cross-AZ traffic charges are incurred when the Data Migration job collects data from the source database.

    ![Cross-AZ traffic charges](/media/tidb-cloud/dm-billing-cross-az-fees.png)

- If the Data Migration job and the TiDB cluster are not in the same AZ, cross-AZ traffic charges are incurred when the Data Migration job writes data to the target TiDB cluster.

    ![Cross-region and cross-AZ traffic charges](/media/tidb-cloud/dm-billing-cross-region-and-az-fees.png)

The cross-region and cross-AZ traffic prices are the same as those for TiDB Cloud. For more information, see [TiDB Cloud Pricing Details](https://en.pingcap.com/tidb-cloud-pricing-details/).

> 

Author: ```suggestion

If the data between the upstream database and the Data Migration job is incoming through the public network, TiDB Cloud will not charge for the traffic, regardless of whether the upstream database and the Data Migration job are across regions or across AZs.

```

The Data Migration job measures incremental data migration performance in rows/s. This unit indicates the number of rows that are migrated to the target database per second. For example, if the upstream database executes `INSERT`, `UPDATE`, or `DELETE` statements of 10,000 rows in about 1 second, the Data Migration job of the corresponding specification can replicate the 10,000 rows to the downstream in about 1 second.

## Price

The following table lists the price of TiDB Cloud for each Data Migration RCU:

| Cloud provider | Region                      | RCU price ($/hr) |
|----------------|-----------------------------|------------------|
| AWS            | Oregon (us-west-2)          |          $0.1307 |
| AWS            | N. Virginia (us-east-1)     |          $0.1307 |
| AWS            | Mumbai (ap-south-1)         |          $0.1393 |
| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
| AWS            | Frankfurt (eu-central-1)    |          $0.1564 |

The Data Migration job is in the same region as the target TiDB cluster.

> Note that if the source database and the TiDB cluster are not in the same region or not in the same availability zone (AZ), two additional traffic charges will be incurred: cross-region and cross-AZ traffic charges.

Author: ```suggestion
Note that if you are using Private Endpoint connections or VPC Peering, and if the source database and the TiDB cluster are not in the same region or not in the same availability zone (AZ), two additional traffic charges will be incurred: cross-region and cross-AZ traffic charges.
```

- If the source database and the TiDB cluster are not in the same region, cross-region traffic charges are incurred when the Data Migration job collects data from the source database.

    ![Cross-region traffic charges](/media/tidb-cloud/dm-billing-cross-region-fees.png)

- If the source database and the TiDB cluster are in the same region but in different AZs, cross-AZ traffic charges are incurred when the Data Migration job collects data from the source database.

    ![Cross-AZ traffic charges](/media/tidb-cloud/dm-billing-cross-az-fees.png)

- If the Data Migration job and the TiDB cluster are not in the same AZ, cross-AZ traffic charges are incurred when the Data Migration job writes data to the target TiDB cluster.

    ![Cross-region and cross-AZ traffic charges](/media/tidb-cloud/dm-billing-cross-region-and-az-fees.png)

The cross-region and cross-AZ traffic prices are the same as those for TiDB Cloud. For more information, see [TiDB Cloud Pricing Details](https://en.pingcap.com/tidb-cloud-pricing-details/).

> If the data between the upstream database and the Data Migration job is incoming through the public network, TiDB Cloud will not charge for the traffic, regardless of whether the upstream database and the Data Migration job are across regions or across AZs.

Author: ```suggestion
If the data between the upstream database and the Data Migration job is through the public internet, TiDB Cloud will not charge for the traffic, regardless of whether the upstream database and the Data Migration job are across regions or across AZs.
```
